src/modules/lazy/lazy.less

```diff
@@ -14,13 +14,17 @@
   margin-top: -21px;
   z-index: 10;
   transform-origin: 50%;
-  animation: swiper-preloader-spin 1s infinite linear;
+  animation: none;
```

Comment:
Wouldn't just removing `animation` completely work the same way?

Reply:
Yes of course